src/_pytest/config/__init__.py

```diff
@@ -540,11 +540,7 @@ def _is_in_confcutdir(self, path: Path) -> bool:
         """
         if self._confcutdir is None:
             return True
-        try:
-            path.relative_to(self._confcutdir)
-        except ValueError:
-            return False
-        return True
+        return path not in self._confcutdir.parents
```

Comment:
i recall parents being painfully expensive,

presuming the given paths are absolute, would it make sense to ensure that the path parts match (aka the common path between path and confcutdir being the path)

this is a idea for a follow-up unless its a easy check